lib/angular-evaporate.js

```diff
@@ -147,6 +157,11 @@
                       (100 - that.$percent) / that.$percent *
                       (now - that.$started) / 1000
                     );
+                    var progress = (arg * 100).toFixed(2);
+                    var delta = (new Date() - that.$started) / 1000;
+                    var loaded = (progress * that.sizeBytes) / 100;
```

Comment:
[`File.sizeBytes`](https://developer.mozilla.org/en-US/docs/Web/API/File/fileSize) is non-standard and moreover it's deprecated, so please, use [`Blob.size`](https://developer.mozilla.org/en-US/docs/Web/API/Blob/size) instead